packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeLink/MT5TradeLink.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react';
+import { WalletButton, WalletText } from '../../../../../components/Base';
+import MT5Icon from '../../../../../public/images/ic-mt5.svg';
+import WindowsIcon from '../../../../../public/images/ic-windows-logo.svg';
+import MacOSIcon from '../../../../../public/images/ic-macos-logo.svg';
+import LinuxIcon from '../../../../../public/images/ic-linux-logo.svg';
+import './MT5TradeLink.scss';
+
+const AppToIconMapper = {
+    linux: <LinuxIcon />,
+    macos: <MacOSIcon />,
+    web: <MT5Icon />,
+    windows: <WindowsIcon />,
+};
+
+const AppToTitleMapper = {
+    linux: 'MetaTrader 5 Linux app',
+    macos: 'MetaTrader 5 MacOS app',
+    web: 'MetaTrader 5 web',
+    windows: 'MetaTrader 5 Windows app',
+};
+
+const AppToButtonTextMapper = {
+    linux: 'Learn more',
+    macos: 'Download',
+    web: 'Open',
+    windows: 'Download',
+};
```

Comment:
```suggestion
```
All these mapper object, since they share the same key can be combined as suggested below:

```
const AppDataMapper = {
    linux: {
        titile: 'MetaTrader 5 Linux app',
        icon: <LinuxIcon />,
        buttonText: 'Learn more'
    }, 
    ....
}
```